odbc/src/api/functions.rs

```diff
@@ -384,7 +384,18 @@ pub unsafe extern "C" fn SQLBindCol(
                 return SqlReturn::ERROR;
             }
 
-            let max_col_index = mongo_stmt.as_ref().unwrap().get_resultset_metadata().len();
+            let max_string_length = *(*stmt.connection)
```

Comment:
Looking back now, I redo this same thing all over the place, so I'll extract this to a helper.

Reply:
I'd like to see this helper made if possible!